iml-manager-cli/src/main.rs

```diff
@@ -129,17 +129,24 @@ fn start_spinner(msg: &str) -> impl FnOnce(Option<String>) -> () {
     }
 }
 
-fn display_cmd_state(cmd: &Command) {
+fn display_cmd_state(cmd: &Result<Command, iml_manager_client::ImlManagerClientError>) {
```

Comment:
This fn shouldn't be changed, A cmd is not going to be wrapped in an Error.

Reply:
Maybe I didn't understand. The previous comment said to refactor:
> Instead of constructing a virtual command just to display the message, you should refactor display_cmd_state so that it calls different message fns for success / failure etc...

Reply:
I reverted this function and created a new function to handle the validation error case.

Reply:
Instead of reusing `display_cmd_state` for things not command related, take the `println`s and make separate functions out of them:

`display_error(...)`, `display_success(...)`

Then when handling an API error that's not related to a command, you can call those fns directly and not conflate that error with a command.